shopinvader/services/cart.py

```diff
@@ -45,9 +45,16 @@ def update(self, **params):
             return self._to_json(cart)
 
     def add_item(self, **params):
-        cart = self._get()
+        cart = self._get(create_if_not_found=False)
+        # Cart default params could be passed prefixed w/ `cart__`
+        params, cart_params = self._extract_cart_params(**params)
         if not cart:
-            cart = self._create_empty_cart()
+            cart = self._create_empty_cart(**cart_params)
+        # TODO: shall we update cart params before anyway?
```

Comment:
@lmignon @sebastienbeau I'd like to hear your voice about this if possible :pray: